charts/castai-db-optimizer/templates/pgcat-config.yaml

```
@@ -2,18 +2,35 @@
 {{- if not .Values.pooling.databases }}
 {{- fail "pooling.databases cannot be empty when pooling is enabled. Please provide at least one database name." }}
 {{- end }}
+{{- $secretName := "" }}
+{{- if .Values.pooling.credentialsSecretRef }}
+{{- $secretName = .Values.pooling.credentialsSecretRef.name }}
```

The `name` seems redundant.

```suggestion
{{- $secretName = .Values.pooling.credentialsSecretRef}}
```